### ISSUE TITLE
[feat] allow to add and assign tags in the tags view

### ISSUE THREAD

**Issue by umlaeute** [raimund-schluessler/inventory#535]
afaict, the only option to add a new tag is by specifying it when creating a new item (via the CSV import).

when editing an existing item, I *can* type a new tag into the tags field (and it gets displayed as <kbd>mytag</kbd>), but the pulldown also says:
> No tag available. Create one!

the tag is lost as soon as i leave the editor.

in the `tags` menu, i cannot add any tag at all.

**Comment by raimund-schluessler** [comment]
When you edit an item, the tags do get saved (and created, if the do not exisit). But the problem is, that the edits are not pushed to the server, if you just click outside. That is a bug, I filed https://github.com/raimund-schluessler/inventory/issues/536 for this.
If you use the button (the checkmark at the bottom right), the edits do get saved and the new tag gets created.

**Comment by raimund-schluessler** [comment]
Adding a tag in the tags menu would not really make sense, as it is a filter only.

**Comment by raimund-schluessler** [comment]
> i think it would be nice if _Inventory_ could follow that model.

Fair enough. But that would require that the app allows to edit multiple items at once first. So that would require #8 before, I think.
